### DISCUSSION TITLE
i already use 117. how to update to 119?

### DISCUSSION THREAD

**Discussion by validMyID** [Alex313031/thorium#493] [Q&A] [answered]
hello

please help me to update my 117 to 119
already download the zip but cannot find the exe

**Comment by validMyID** [comment]
i found from winget

 https://github.com/Alex313031/Thorium-Win/releases/download/M119.0.6045.214/thorium_mini_installer.exe

correct?
@Alex313031

**Reply by gz83** [reply, marked as answer]
yeah

@validMyID
